test/e2e/containers/prow-npd-e2e/script.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This scripts clones a NPD repository, and then runs NPD e2e test from it.
+
+set -o errexit
+set -o pipefail
+set -u
+set -x
+
+test() {
+  git clone https://github.com/kubernetes/node-problem-detector.git
+
+  pushd node-problem-detector
+
+  ZONE=us-central1-a PROJECT=xueweiz-experimental \
```

Comment:
@krzyzacy Hi Sen, I would prefer to use the "Prow managed projects" for this purpose.
In that case, do you know how should I set up the ZONE and PROJECT environment variables? Thanks!